.github/workflows/publish.yml

```diff
@@ -93,7 +93,8 @@ jobs:
       - name: Calculate release name
         id: release-name
         run: |
-          echo "release_version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_OUTPUT
+          TAG="${GITHUB_REF/refs\/tags\//}"
+          echo "release_version=${TAG#v}" >> $GITHUB_OUTPUT
```

Comment:
(Presumably you could also do `release_version=${GITHUB_REF/refs\/tags\/v/}`, but this is clearer :) )

Reply:
My concern with this was that it assumes the `v` is always present, doing in two steps ensures the `/refs/tags/` is always removed ... could have made the `v` and optional capture I suppose 🤦🏻 I'll leave as is for the mo, it can be iterated on.